CallTimeCounter/CallTimeCounter.plugin.js

```diff
@@ -115,7 +115,8 @@ module.exports = !global.ZeresPluginLibrary ? class {
         }
 
         render() {
-            return React.createElement("div", { className: "voiceTimer" }, `Time elapsed: ${new Date(this.state.delta).toISOString().substr(11, 8)}`);
+            var hrs=parseInt(this.state.delta/86400)*24;
+            return React.createElement("div", { className: "voiceTimer" }, `Time elapsed: ${(new Date(this.state.delta%86400*1000)).toUTCString().replace(/.*(\d{2}):(\d{2}):(\d{2}).*/, function replacer(match, p1, p2, p3, offset, string, d=hrs){return [+p1+d, p2, p3].join(':');})}`);
```

Comment:
Wouldn't it at this point just make more sense to construct the HH:MM:SS string ourselves? Feels like a way cleaner solution than this

Reply:
Yeah, you're right. I figured using a built in method might have been cleaner but that was before I wrote this monstrosity to work around it's quirks 😅